core/src/main/java/org/opensearch/sql/data/model/ExprDatetimeValue.java

```diff
@@ -1,99 +1 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
 
```

Comment:
Just want to confirm, this file is completely removed by this PR and not just made to have a single empty line

Reply:
This file is completely removed, for some reason the file wasn't actually deleted though, it has now been updated